routes/api/user/read-user.js

```diff
@@ -6,6 +6,23 @@ var db = require('../../../models');
 var router = express.Router();
 
 console.log('read-user');
+app.get("/api/users", function(req, res) {
+    // 1. Add a join to include all of users
+    db.User.findAll({}).then(function(dbUser) {
+      res.json(dbUser);
+    });
+  });
+
+  app.get("/api/users/:id", function(req, res) {
+    // 2; Add a join to include all of users
+    db.Author.findOne({
+      where: {
+        id: req.params.id
+      }
+    }).then(function(dbUser) {
+      res.json(dbUser);
+    });
+  });
```

Comment:
Let's add a `router.get` for `/api/users/:id/services` which makes a `db.Users.findOne` call `where` the `id` is equal to the request params id and then on the result, it makes a `request.getServices()` call and returns the results as json. This uses the automatically-generated get association method. See: https://sequelize.org/master/class/lib/associations/belongs-to-many.js~BelongsToMany.html